data/human/ships.txt

```diff
@@ -885,53 +889,59 @@ ship "Carrier"
 		"fuel capacity" 700
 		"cargo space" 100
 		"outfit space" 820
-		"weapon capacity" 370
+		"weapon capacity" 240
 		"engine capacity" 210
 		weapon
 			"blast radius" 300
 			"shield damage" 3000
 			"hull damage" 1500
 			"hit force" 4500
 	outfits
-		"Particle Cannon" 4
-		"Meteor Missile Launcher" 4
-		"Meteor Missile" 120
-		"Heavy Laser Turret" 2
+		"Meteor Missile Launcher" 2
+		"Meteor Missile" 60
+		"Sidewinder Missile Rack" 8
+		"Sidewinder Missile" 184
+		"Torpedo Launcher" 2
+		"Torpedo" 60
+		"Heavy Laser Turret"
```

Comment:
I think it might be a problem that once the fighters and missiles run out, there's only one continuous-fire offensive turret.
In big battles during FW, it would be possible to let your allies get the small Navy ships and eliminate the missiles.
They'll get exploded, but a two Heavy Laser Turrets is no match for a properly-equipped player.

Reply:
I actually think it's a good thing rather than a problem that the ship has such a weakness.
I don't believe it should be a powerful close-quarters combatant on top of its powerful fighters and missiles.